README.rst

```diff
@@ -242,6 +242,43 @@ Run qa tests:
 
     ./run-qa-checks
 
+Quickstart Guide
+----------------
+
+Creating a topology
+^^^^^^^^^^^^^^^^^^^
+
+1. Create a topology object by going to *Network Topology* > *Topologies*
+   > *Add topology*.
+2. Give an appropriate label to the topology.
+3. Select the *topology format* from the dropdown menu. The *topology format*
+   determines which parser should be used to process topology data.
+4. Select the *Strategy* for updating this topology.
+
+   - If you are using `FETCH strategy <#fetch-strategy>`_, then enter the
+     URL for fetching topology data in the *Url* field.
+   - If you are using `RECEIVE strategy <#receive-strategy>`_, you will get the
+     *URL* for sending topology data. The *RECEIVE* strategy provides an
+     additional field *expiration time*. This can be used to add delay in
+     marking missing links as down.
+
+.. image:: https://github.com/openwisp/openwisp-network-topology/raw/docs/docs/quickstart-topology.gif
+
+Sending data for topology with RECEIVE strategy
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+1. Copy the *URL* generated by OpenWISP for sending the topology data.
```

Comment:
here we can say something like, for example, in our case the URL is `http://127.0.0.1:8000/api/v1/network-topology/topology/d17e539a-1793-4be2-80a4-c305eca64fd8/receive/?key=cMGsvio8q0L0BGLd5twiFHQOqIEKI423`